src/server/shared/Networking/Socket.h

```diff
@@ -34,12 +35,24 @@ using boost::asio::ip::tcp;
 #define TC_SOCKET_USE_IOCP
 #endif
 
+enum ProxyHeaderReadingState {
```

Comment:
we do not inline brackets. Please move them into a new line. Apply this to all the other changes you made.